tutorials/translation/ct2_hindi_translation.ipynb

```diff
@@ -0,0 +1,562 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Indic Translation\n",
+    "This notebook demonstrate an example use of nemo-curator for Indic language generation via translation from English language. This workflow is accelarated by CrossFit, a library that leverages intellegent batching and RAPIDS to accelerate the offline inference on large datasets. \n",
+    "This example uses ctransalte2 model from [here](https://indictrans2-public.objectstore.e2enetworks.net/it2_preprint_ckpts/en-indic-preprint.zip), taken from IndicTrans2 github repo, [here](https://github.com/AI4Bharat/IndicTrans2/tree/main?tab=readme-ov-file#multilingual-translation-models)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ[\"DASK_DATAFRAME__QUERY_PLANNING\"] = \"False\"\n",
+    "import argparse\n",
+    "import re\n",
+    "import time\n",
+    "from dataclasses import dataclass\n",
+    "from functools import lru_cache\n",
+    "\n",
+    "import cudf\n",
+    "import ctranslate2\n",
+    "import numpy as np\n",
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "from crossfit import op\n",
+    "from crossfit.backend.torch.hf.model import HFModel\n",
+    "from dask.distributed import get_worker\n",
+    "from nltk.tokenize import sent_tokenize\n",
+    "from transformers import AutoConfig, AutoModelForSeq2SeqLM, AutoTokenizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo_curator.classifiers.base import DistributedDataClassifier\n",
+    "from nemo_curator.datasets import DocumentDataset\n",
+    "from nemo_curator.utils.distributed_utils import get_client, load_object_on_worker\n",
+    "from nemo_curator.utils.script_utils import ArgumentHelper"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For pre and post processing, we are using IndicTransToolkit, required for translation using IndcTrans2 models. It is simple, modoular library for preprocessing, normalizations, postprocessing stuff."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    from IndicTransToolkit import IndicProcessor\n",
+    "except ImportError:\n",
+    "    raise ImportError(\n",
+    "        \"IndicTransToolkit not found. Please install it using the following command: \\n\"\n",
+    "        + \"pip install git+https://github.com/VarunGumma/IndicTransToolkit.git\"\n",
+    "    )\n",
+    "import dask_cudf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
Add a table of contents here. 

See https://github.com/NVIDIA/NeMo-Curator/blob/cef1c3408684e9ce96e8125f041378407daf0a7b/tutorials/image-curation/image-curation.ipynb example for input